unitOneAssessmentPractice.js

```diff
@@ -3,40 +3,95 @@ let assert = require('assert')
 // Question One:
 
 // Write a function called double that doubles a number
+const double = num => num*2
+
 
 // Uncomment out the next line to test your solution
 // runQ1Tests()
 
 // Question Two:
 
 // Write a function called containsSeven that returns whether or not a number has an sevens in it.
+const containsSeven = num =>{
+  let number = String(num);
+  if(number.includes("7")){
```

Comment:
Since the includes method returns a boolean, you can just return `return number.include("7")`